configs/sites/tier1/hera/mirrors.yaml

```diff
@@ -1,15 +1,15 @@
 mirrors:
   local-source:
     fetch:
-      url: file:///contrib/spack-stack/mirror
+      url: file:///scratch1/NCEPDEV/nems/role.epic/spack-stack/mirror-1.8.0
```

Comment:
Do the mirror paths need to be version-specific? Why not use one source cache across versions so we don't have to redownload for each release?

Reply:
I thought mirror is containing particular versions of apps, which differ from release to release?

Reply:
mirrors are not version specific, should always point to the one authoritative mirr

Reply:
Yes, but cached packages are all differentiated by sha256 or commit hash, so I don't think there's any downside to putting them all in one big pot (and then we get the benefit of minimizing disk space and saving time by reusing those packages that don't change between versions). It's not a huge deal since in the future the source cache on each system will get populated by automated builds. Anyway, I'll approve the PR and it's up to you how you want to handle the source cache paths.

Reply:
I'll make change.